### ISSUE TITLE
Loading ready event

### ISSUE THREAD

**Issue by Jehutty** [chrisrzhou/react-globe#2]
Hi there, first of all excellent work with creating such an interactive, smooth and powerful visualisation. 

I am trying to use the globe with a 16k jpg texture of the earth so I can have more detail in the texture as to when i zoom in.

The globe rendering takes a while though ( couple of seconds ) when the component mounts but I currently have no way of displaying a loader/mesh placeholder before the initial rendering is complete. Any insight to this?

**Comment by chrisrzhou** [comment]
Hi thanks for checking this out!  The library currently doesn't handle rendering a loader/mesh placeholder.  I would be happy to add this in as an option in `globeOptions`.  I don't really know how to do this in ThreeJS (I can explore and check out how to do this), but if you have a really simple example that works in Codesandbox.io, I can look at it and add this in.

Or feel free to submit a PR if you know how to add this in!

**Comment by Jehutty** [comment]
Hey thanks for the quick reply!
 I am currently working on finding out how to do it.

 Three JS supports this feature here https://threejs.org/docs/#api/en/loaders/managers/LoadingManager

 I am not familiar with your library as to how its constructed or TS though, maybe you will be able to solve it. In the meantime I am trying to find a simple workaround for it with some sort of HOC or conditional rendering with react redux.

**Comment by chrisrzhou** [comment]
Awesome, I might have time to get to this next week and maybe support loader/mesh placeholders for both globe and backgrounds!

**Comment by chrisrzhou** [comment]
Hi @Jehutty, sorry I wasn't able to get to this earlier.  I included this enhancement in 3.1.1 ([codesandbox example](https://codesandbox.io/s/react-globe-311-cdp92)).  The docs in `Textures` also cover this in detail.

tldr;  I exposed a `onTextureLoaded` method that will allow you to pass a callback when the texture successfully load.  A few things I concerned while working on this:
- I looked at `LoadingManager` and I did not want to support all the functionality `start`, `error`, `loaded` and increase the complexity of the component.  I think you bring a valuable point about having some control in determining if the component is loading.  If you look at the codesandbox example, you can see how you can achieve the effect outside of the component with the help of `onTextureLoaded` and React hooks.
- This only applies to the globe texture and does not work for background and cloud textures.  In the future, I can include that functionality as well, but I'll leave it to the future to add it if more people would like that control.  I still want to prioritize keeping the component as simple as possible!
